WPMedia/arc-themes-components
# File Naming Update

## Description

Update inconsistency of file naming to follow existing pattern of index

## Test Steps

1. Checkout branch - `git checkout update-files-to-use-index`
2. Update dependencies - `npm i`
3. Run Storybook `npm run storybook`
4. Check out the storybook

Tests in action should cover the changes

## Author Checklist 


## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._
